src/ekat/util/ekat_lin_interp.hpp

```diff
@@ -87,6 +87,15 @@ struct LinInterp
                   const V3& y1,
                   const V4& y2) const;
 
+  template <typename V1, typename V2, typename V3, typename V4, typename RangePolicy>
+  KOKKOS_INLINE_FUNCTION
+  void lin_interp(const MemberType& team,
+                  const RangePolicy& range_policy,
```

Comment:
I think `RangePolicy` might be a confusing name, leading someone to confuse team vs range policies (in the kokkos sense). Perhaps we could call the typename `RangeBoundariesStruct`. After all, in Kokkos, `TeamThreadRange(..)` and `ThreadVectorRange(..)` both return an `XYZRangeBoundaryStruct`, with `XYZ=TeamThread` and `XYZ=ThreadVector` respectively. Also, these structs are not reall "policies", per se...

Reply:
Agree.